src/routes/api/chat/+server.ts

```diff
@@ -1,6 +1,7 @@
 // Defined in a `$REPO/.env` file as described in https://kit.svelte.dev/docs/modules#$env-static-private
 import { CHAT_API_KEY } from '$env/static/private';
 import type { Message, ChatRequest } from '$lib/types';
+import { CohereBackend, OpenAIBackend } from '$lib/ai';
```

Comment:
Is this used? (if not - delete?)

Reply:
sure thing! I was stubbing out the imports for the refeactoring step of pulling out the hardcoded call. I'm totally with good with stripping it out here and adding it back in. Agree that adding it in this pr isn't pretty / neat.